ctapipe/io/utils.py

```diff
@@ -1,6 +1,8 @@
 import gzip
 
-
+# This function is currently needed by some EventSource, which will soon
+# be removed and converted into plugins.
+# Then this can be removed from ctapipe
 def is_fits_in_header(file_path):
```

Comment:
This function does not really test for fits compatiblity. `FITS` is not a required keyword. However, the FITS standard clearly states:

> The SIMPLE keyword  is required to  be  the  first  keyword in the primary header of all FITS files.

So, we only need to read the first 6 bytes and check if they are `b'SIMPLE'`

Reply:
nice.

Reply:
should also probably rename it to `is_fits_format()` or something like that, if you are no longer testing for "fits" in the header.